kubernetes-manifests/balance-reader.yaml

```diff
@@ -51,7 +51,7 @@ spec:
           value: "1000000"
         # tell Java to obey container memory limits
         - name: JVM_OPTS
-          value: "-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Xms128m -Xmx256m"
+          value: "-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Xms256m -Xmx512m"
```

Comment:
Any issue discovered with these Memory/CPU limit updates? With ASM or something?

Reply:
Some time ago I very liberally tuned down all of the resource limits everywhere to reduce the amount of GKE nodes necessary to deploy (from 4 to 3). I'm slowly tuning it as I discover what bottlenecks and what we can tune back up without reaching the "4 nodes minimum" amount. This memory bump helped a lot with the startup time (Java Spring processes can have very bulky boot up sequences)